src/lang/en-US.json

```diff
@@ -620,6 +621,7 @@
   "rune_6": "Arcane",
 
   "search_title": "Player Search",
+  "search_title_2": "Player, Match Search",
```

Comment:
probably "Search by player name, match ID..."

Reply:
Any reason you opted to make a new string instead of replacing the existing one?